### PR TITLE
Bugfix/update setup and readme for GitHub

### PR DESCRIPTION
**Pull request by phargogh**
This PR corrects links to bitbucket throughout the source tree so that they now point to their new homes on github.

### CODE REVIEW

File: src/natcap/invest/ui/model.py [hunk 1]
```diff
@@ -560,7 +560,7 @@ def __init__(self, parent=None):
                 ('numpy', 'BSD', 'http://numpy.org'),
                 ('pyamg', 'BSD', 'http://github.com/pyamg/pyamg'),
                 ('pygeoprocessing', 'BSD',
-                 'http://bitbucket.org/richsharp/pygeoprocessing'),
+                 'http://github.com/natcap/pygeoprocessing'),
```
Comment (dcdenu4) [inline src/natcap/invest/ui/model.py]: Should this be http[s]?

Comment (phargogh) [reply src/natcap/invest/ui/model.py]: Yes indeed!  Patched in 0ab5458.

**Comment by phargogh** [comment]
@dcdenu4 thanks for taking a look!  I've pushed an updated comment and have resolved a merge conflict on `HISTORY.rst`.  Back to you!

**Comment by dcdenu4** [comment]
@phargogh Looks like there might be some fresh HISTORY.rst conflicts. I approved the PR but I should probably let you handle the conflict.

**Comment by phargogh** [comment]
Thanks @dcdenu4 !  I've resolved the conflict and am just waiting on the tests to finish before merging.

**Comment by dcdenu4** [comment]
Come onnnnnn Build InVEST App Binaries / Build windows binaries ... come onnnn

**Comment by phargogh** [comment]
There we go!

**Pull request merged**